### PR TITLE
Rewrite fitters as separate files. Pydocstyle (pep257) is added.

### PR DESCRIPTION
**Pull request by aprotopopov**
Short summary:
- All fitters now in lifetimes/fitters. (*major change*)
- Pydocstyle and pycodestyle with configs in setup.py are added for dev_requirements.txt
- May be some ignoring for now errors for pydocstyle should be uncommented and fixed (like D100 and D104)
- Added two make receipts for pycodestyle and pydocstyle

P.S. What about docs?

### CODE REVIEW

**Comment by CamDavidsonPilon** [comment]
Nice - this resembles `lifelines` now (and will make docs easier)

Sorry I didn't get to the docs this weekend, I will try again this week. Feel free start a branch though, don't wait for me.

**Comment by CamDavidsonPilon** [comment]
So I merged https://github.com/CamDavidsonPilon/lifetimes/pull/113 first, as I think that PR would have been harder to rebase. Can you rebase this please?

File: lifetimes/estimation.py [hunk 1]
```diff
@@ -1,906 +1,11 @@
...
-from collections import OrderedDict
+"""All fitters from fitters directory."""
 
-import numpy as np
-import pandas as pd
-from numpy import log, exp, logaddexp, asarray, any as npany, c_ as vconcat,\
-    isinf, isnan, ones_like
-from pandas import DataFrame
-import dill
+from .fitters import BaseFitter
```
Comment (CamDavidsonPilon) [inline lifetimes/estimation.py]: 👍

File: setup.cfg [hunk 2]
```diff
@@ -2,4 +2,26 @@
 ignore=E501,E241
 
 [flake8]
-ignore=E501,E241
```
Comment (CamDavidsonPilon) [inline setup.cfg]: 😍 yes this is a great QOL improvement

**Comment by aprotopopov** [comment]
PR #113 is merged.

**Pull request merged**